instrumentation/otel-webserver-module/test/ApacheTesting.sh

```diff
@@ -66,6 +72,10 @@ if [ $targetSystem = "ubuntu" ]; then
 	targetConfFile='/etc/apache2/opentelemetry_module.conf'
 fi
 
+if [ $targetSystem = "centos7" ]; then
```

Comment:
Should we use elif here instead? This makes it more clearer

Reply:
done